src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+#if NETCOREAPP
```

Comment:
Why is this needed?

Reply:
This is causing the failure `src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs(89,40): error CS0103: The name 'ArrayPool' does not exist in the current context`.

Reply:
I pushed the fix.